yarn-project/simulator/src/avm/avm_tree.test.ts

```diff
@@ -326,6 +327,65 @@ describe('Simple Nullifier Consistency', () => {
     // Check insertion results - note we can only compare against the post-insertion results
     checkEqualityOfInsertionResults(containerInsertionResults, worldStateInsertionResults.slice(preInsertIndex));
   });
+
+  it('Should check that the insertion paths resolve to the root', async () => {
```

Comment:
Does this test make sense after my changes that make getLeafOrLowLeafInfo never mutate the tree?

Reply:
Yeah i think this would still work. This checks that i can get to a `final_root` from an `initial_root` via the InsertionResults - which is good sanity check